Libplanet/Net/Swarm.cs

```diff
@@ -305,6 +310,8 @@ public void Dispose()
         /// the dial-up is cancelled after this timeout, and it tries another peer.
         /// If <c>null</c> is given it never gives up dial-ups.
         /// </param>
+        /// <param name="broadcastBlockInterval">The time period of broadcasting chain tip.
+        /// </param>
         /// <param name="broadcastTxInterval">The time period of exchange of staged transactions.
         /// </param>
```

Comment:
```suggestion
        /// <param name="broadcastBlockInterval">Time interval between each broadcast of
        /// chain tip.</param>
        /// <param name="broadcastTxInterval">Time interval between each broadcast of staged
        /// transactions.</param>
```

Just a minor clarification.